### DISCUSSION TITLE
HOW TO MIGRATE FROM NEXTAUTH.JS TO AUTH.JS

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [nextauthjs/next-auth#9169] [Help]
Are there any migration guides to migrate from NextAuth.js to Auth.js?

Changing imports from everything related to `@next-auth` to `@auth` results in errors.
We've been using NextAuth with Prisma adapter for 2 years and now we must migrate since Tiktok isn't supported on NextAuth but supported on Auth.. Kinda random..

For example:
![Screenshot 2023-11-17 at 09 14 42](https://github.com/nextauthjs/next-auth/assets/22228431/7da82e16-c30f-4775-9de0-a9e66d6a4cf0)

Returns errors:
![Screenshot 2023-11-17 at 09 14 50](https://github.com/nextauthjs/next-auth/assets/22228431/aeacee50-3fb6-4de8-985c-d86539f8aa19)

**Comment by DewinU** [comment]
They stopped adding new providers to next-auth@v4
